lib/puma/server.rb

```diff
@@ -386,11 +386,11 @@ def handle_servers
           @reactor.shutdown
         end
       rescue Exception => e
-        STDERR.puts "Exception handling servers: #{e.message} (#{e.class})"
-        STDERR.puts e.backtrace
+        # STDERR.puts "Exception handling servers: #{e.message} (#{e.class})"
```

Comment:
[Raises as if STDERR was closed](https://travis-ci.org/puma/puma/jobs/237251772#L453) if not commented out.